### PR TITLE
Mat 7414 add number of saved codes

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-7414](https://jira.cms.gov/browse/MAT-7414)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/CqlBuilderPanel/codesSection/CodesSection.tsx [hunk 1]
```diff
@@ -69,6 +70,7 @@ export default function CodesSection({
           };
         });
         setParsedCodesList(codesList);
+        setSavedCodes(parsedCodesList ? parsedCodesList?.length : 0);
```
Comment (adongare) [inline src/CqlBuilderPanel/codesSection/CodesSection.tsx]: we are using `parsedCodesList`, I wonder if we could get rid of line 42 and its usage

Comment (sb-cecilialiu) [reply src/CqlBuilderPanel/codesSection/CodesSection.tsx]: @adongare I refined the code and used parsedCodesList instead of a new state variable, thanks Ashok!

**Comment by sb-cecilialiu** [comment]
Closing this PR after talking with Ashok because once the refresh happens after delete, then parsedCodesList got refreshed so there is no need to pass it to SavedCodesSubSection.

**Comment by sb-cecilialiu** [comment]
Reopen this PR as issue still exists on dev.

**Review by adongare — APPROVED** [review]
